docker-compose.yml

```diff
@@ -20,7 +22,7 @@ services:
       - anvil
         --port=8545
         --host=0.0.0.0
-        --accounts=${NUM_NWAKU_NODES:-5}
+        --accounts=120
```

Comment:
Can we use more to make sure there wont be a collision? waku-simulator can technically run with up to 200ish nodes. So maybe generate 520 and take the last 20.

Assuming generating 520 wont take lots of time. If it does we can reconsider.

Reply:
It looks like generating 520 doesn't take long at all. 
Updated in https://github.com/waku-org/waku-simulator/pull/77/commits/1c05af8f0392decdb5daa47a33fde1e9dd8603f8